sdk/java-sdk/src/main/scala/kalix/javasdk/impl/action/ActionEffectImpl.scala

```diff
@@ -58,6 +56,13 @@ object ActionEffectImpl {
     protected def withSideEffects(sideEffects: Seq[SideEffect]): ForwardEffect[T] =
       copy(internalSideEffects = sideEffects)
   }
+  final case class IgnoreEffect[T](internalSideEffects: Seq[SideEffect]) extends PrimaryEffect[T] {
```

Comment:
Is the `internalSideEffects` still needed when not allowed?

Reply:
it is because is a PrimaryEffect